docs/ldml/tr35.md

```diff
@@ -2,17 +2,17 @@
 
 # Unicode Locale Data Markup Language (LDML)
 
-|Version|48 (draft)|
+|Version|49 (draft)|
 |-------|----------|
 |Editors|Mark Davis (<a href="mailto:markdavis@google.com">markdavis@google.com</a>) and <a href="tr35-acknowledgments.md#acknowledgments">other CLDR committee members</a>|
-|Date|2025-10-11|
-|This Version|<a href="https://www.unicode.org/reports/tr35/tr35-76/tr35.html">https://www.unicode.org/reports/tr35/tr35-76/tr35.html</a>|
-|Previous Version|<a href="https://www.unicode.org/reports/tr35/tr35-75/tr35.html">https://www.unicode.org/reports/tr35/tr35-75/tr35.html</a>|
+|Date|2025-10-30|
+|This Version|<a href="https://www.unicode.org/reports/tr35/tr35-76/tr35.html">https://www.unicode.org/reports/tr35/tr35-77/tr35.html</a>|
+|Previous Version|<a href="https://www.unicode.org/reports/tr35/tr35-75/tr35.html">https://www.unicode.org/reports/tr35/tr35-76/tr35.html</a>|
```

Comment:
In the above two lines, need to also update the actual link, not just the displayed version of it (tr35-nn appears in 2 places for each entry)